app/assets/stylesheets/finder_frontend.scss

```diff
@@ -56,119 +56,86 @@
   margin-top: govuk-spacing(2);
 }
 
-.signup-content p {
-  margin: govuk-spacing(3) 0;
-  @include govuk-font(19);
-
-  @include media(tablet) {
-    margin: govuk-spacing(6) 0;
-    max-width: $two-thirds;
-  }
-}
-
 .metadata-summary {
   @include govuk-font(19);
   margin-bottom: govuk-spacing(6);
 }
 
-#finder-frontend {
-  margin-bottom: govuk-spacing(8);
+.logo {
+  margin: govuk-spacing(6) 0;
+}
 
-  header {
-    .logo {
-      margin: govuk-spacing(6) 0;
-    }
+.logo__image {
+  max-width: 100%;
+}
 
-    #logo-image{
-      max-width: 100%
-    }
+.related-links {
+  margin-top: govuk-spacing(8);
+  margin-bottom: govuk-spacing(8);
+}
 
-    .related-links {
-      @include govuk-font(16, $weight: bold);
-      margin-top: govuk-spacing(8);
-      margin-bottom: govuk-spacing(8);
-      list-style:none;
+.related-links__item {
+  margin-bottom: 4px;
+  list-style: none;
 
-      li {
-        margin-bottom:4px;
-        @include media(tablet) {
-          margin-bottom:8px;
-        }
-
-        a:link,
-        a:visited {
-          text-decoration: none;
-        }
-        a:hover {
-          text-decoration: underline;
-        }
-      }
-    }
+  @include media(tablet) {
+    margin-bottom: 8px;
   }
+}
 
-  .filter-form {
-    .js-enabled & .button__wrapper {
-      display:none;
-    }
-
-    .js-required {
-      display: none;
-    }
-
-    p {
-      max-width: $full-width;
-      @include govuk-font(16);
-      margin-bottom: govuk-spacing(6);
-    }
+.related-links__link {
+  @include govuk-font(16, $weight: bold);
+  text-decoration: none;
 
-    .govuk-select {
-      width: 100%;
-    }
+  &:hover {
+    text-decoration: underline;
   }
+}
 
-  .help-text {
-    @include govuk-font(14);
-    display:block;
-    padding: govuk-spacing(2)/2 0;
-    @include media(tablet) {
-      padding: govuk-spacing(2) 0;
-    }
+.filter-form {
+  .js-enabled & .button__wrapper {
+    display:none;
   }
 
-  .form {
-    @include govuk-font(19);
-    @include media(tablet) {
-      max-width: $two-thirds;
-    }
+  .js-required {
+    display: none;
+  }
 
-    p {
-      max-width: $full-width;
-    }
+  .govuk-select {
+    width: 100%;
```

Comment:
Would it be better to use the `govuk-!-width-full` class on the element to get the same outcome?

Reply:
I don't think we can use this here as it's applying this style to a component. Select components are the width they are, there's no GOVUK Frontend alternative. We could update the component to include this option, that might be worth doing in the future.